Source/Scene/Expression.js

```diff
@@ -1139,11 +1146,10 @@ define([
                     return 'bool(' + left + ')';
                 } else if (value === 'Number') {
                     return 'float(' + left + ')';
-                } else if (value === 'abs') {
-                    return 'abs(' + left + ')';
-                } else if (value === 'cos') {
-                    return 'cos(' + left + ')';
+                } else if (defined(unaryFunctions[value])) {
+                    return value + '(' + left + ')';
                 }
+
```

Comment:
Remove this empty line so that the `else if` below is attached, even though it is wrapped with a pragma.